_posts/2017-04-04-kubernetes-redirect-http-to-https-with-elb-and-the-nginx-ingress-controller.md

```diff
@@ -0,0 +1,78 @@
+---
+date: 2017-04-04
+title: Kubernetes: Redirect HTTP to HTTPS with ELB and the nginx ingress controller
```

Comment:
You should check if the server starts. Here is a yaml problem (`:` in the title => add quotes I think)